networkit/_NetworKit.pyx

```diff
@@ -1347,6 +1365,23 @@ cdef class Graph:
 			nnodes.insert(node)
 		return Graph().setThis(self._this.subgraphFromNodes(nnodes, includeOutNeighbors, includeInNeighbors))
 
+	def iterNeighbors(self, u):
```

Comment:
Please add support for  directed edges. If for some reason that should be not possible, please do not silently fail.